src/main/java/de/retest/recheck/ui/diff/Alignment.java

```diff
@@ -147,12 +147,30 @@ public static Alignment createAlignment( final Element expectedComponent, final
 				continue;
 			}
 
+			if ( bestMatch.similarity < minimumMatch ) {
+				logger.debug( "Best match {} is below threshold with {} similarity.", bestMatch.element,
+						bestMatch.similarity );
+				alignment.put( expected, null );
+				continue;
+			}
+
 			alignment.put( expected, bestMatch.element );
 			matches.put( bestMatch.element, new Match( bestMatch.similarity, expected ) );
 		}
 		return alignment;
 	}
 
+	private static Double getRequiredMinimumMatch() {
+		final String value = System.getProperty( Properties.ELEMENT_MATCH_THRESHOLD_PROPERTY );
+		try {
+			return Double.parseDouble( value );
+		} catch ( final Exception e ) {
```

Comment:
Check `value` for `null`, then we don't have to catch the generic `Exception` type but only `NumberFormatException`.